src/Plugin/WSConnector/WSConnectorSimpleHTTPWithLangReplacement.php

```diff
@@ -90,7 +99,7 @@ public static function wsconnectorHttpHeaderAjaxCallback(array &$form, FormState
    * {@inheritdoc}
    */
   public function call($options, $method, $replacements = [], $data = NULL, array $tokens = []) {
-    $langcode = $this->language_manager->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
+    $langcode = $options['langcode'] ?? $this->language_manager->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)->getId();
```

Comment:
What's a double question mark do?

Reply:
Null coalescing operator.
It is syntactic sugar for:
$i = isset($a) ? $a : $b;
to be impressed as:
$i ?? $b;

https://www.php.net/manual/en/language.operators.comparison.php#language.operators.comparison.ternary